runtime/test-runtime/src/lib.rs

```diff
@@ -731,3 +758,65 @@ sp_api::impl_runtime_apis! {
 		}
 	}
 }
+
+trait Config {
```

Comment:
What is the purpose of this?

Reply:
:sweat_smile: tbh I have no idea. I copied it from somewhere. I guess I can remove the trait and the impl